blade-graphics/src/gles/command.rs

```diff
@@ -396,6 +398,10 @@ impl crate::traits::RenderPipelineEncoder for super::PipelineEncoder<'_> {
         start_instance: u32,
         instance_count: u32,
     ) {
+        for (c, i) in self.color_targets.iter().zip(0..) {
```

Comment:
We don't want to set this on `draw()`. Instead, the blend states shoud be set when the pipeline is switched